unity-environment/Assets/ML-Agents/Examples/3DBall/Scripts/Ball3DDecision.cs

```diff
@@ -4,30 +4,21 @@
 
 public class Ball3DDecision : MonoBehaviour, Decision
 {
+	public float rotationSpeed = 2f;
```

Comment:
- indentation should be 4 characters, as spaces
- if this is a constant, declare it as public const ROTATION_SPEED and add a comment describing it

Reply:
indentation is fixed. 

Regarding the constant naming convention. 

1. I know in C++ and Java people usually use ROTATION_SPEED as the naming convention. But I went through this post and this post, where people talks about the naming convention in C#. And it seems that microsoft uses RotationSpeed. And they have a tool called StyleCop to check for the coding style in C#. Should we use ROTATION_SPEED or RotationSpeed?
https://stackoverflow.com/questions/242534/c-sharp-naming-convention-for-constants
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/naming-resources

2. Right now in the code base we use rotationSpeed for const, should I change them to one of the above style?

3. In this case, I want the rotation speed to be editable within the unity editor. In that case is this the right way of coding it?

Reply:
Also regarding 1. since the codebase currently is already following a C++ or Java like coding style, maybe we should just keep that, which means we will use ROTATION_SPEED for all the const variable.